doc/devices/dts/flash_partitions.rst

```diff
@@ -227,5 +227,5 @@ See the  `MCUboot documentation`_ for more details on these partitions.
 NFFS Partitions
 ***************
 
-**nffs_partition**
+**storage_partition**
```

Comment:
Is the NFFS Partitions title above this still OK?

Reply:
@dbkinder that's a good point. I've renamed it to "File System Partitions"